### ISSUE TITLE
[ Feature request ] - New way to compare commits

### ISSUE THREAD

**Issue by nfgferreira** [DonJayamanne/gitHistoryVSCode#745]
When you have the log open, it is possible to select only one commit. In order to compare 2 commits you have firstly to select one commit with the More button, go to the other and then choose to compare them with another More option. I think it would be more intuitive and easier to allow the user to select a second commit (with Ctrl-click, for instance). This would automatically show the differences in the bottom pane currently dedicated only to show the list of files that were modified by the commit.

**Comment by socketbox** [comment]
Agreed. The mechanism for comparison is awkward. And doubly so when trying to compare across branches.
